wire-ios/Wire-iOS/Sources/Helpers/SaveFileManager.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import UIKit
+import WireSystem
+
+protocol SaveFileActions {
+    func save(value: String, fileName: String, type: String)
+}
+
+final class SaveFileManager: NSObject, SaveFileActions {
```

Comment:
what do we use it for ? temp folder is clear at launch just FYI

Reply:
That should be fine. We create a temp file to be shared when a user wants to share the E2EI certificate from the details. Used in `DeviceDetailsActionsHandler`. If you note, we also delete this temp file once sharing is complete.